src/TreeView/TreeView.tsx

```diff
@@ -525,7 +533,7 @@ const SubTree: React.FC<TreeViewSubTreeProps> = ({count, state, children}) => {
 
   // Announce when content has loaded
   React.useEffect(() => {
-    if (state === 'done') {
+    if (isPending && state === 'done') {
```

Comment:
One of the problems @jsholes mentioned in the original issue was that announcements were being made with user interaction. Does this change fix that?

I wonder if we should only announce and update when the `state` is `"done"` and the item changes from not-expanded to expanded.

Reply:
Hey @colebemis, assuming that you mean that the announcements you're referring to were the ones about the last child directory when the parent was being expanded, yes, this is a fix for that.

What you're describing sounds like the current behavior for an async folder. The status of "done" isn't mentioned unless the person expands the folder for the first time. Is this what you mean?